pyproject.toml

```diff
@@ -23,23 +18,18 @@ dependencies = [
     "kaleido>=0.2.1",
     "pacmap>=0.7.2",
     "scipy==1.8.1", #pycytominer requires scipy < 1.9
-]
-requires-python = ">=3.9,<3.11"
-
-[project.optional-dependencies]
-viz = [
     "mitsuba>=3.5.0",
-    "plotly>=5.22.0",
-    "seaborn>=0.13.2"
-]
-
-all = [
-    "br[viz]",
+    "seaborn>=0.13.2",
+    "plotly>=5.22.0"
 ]
+requires-python = "==3.10.*"
+license = {text = "MIT"}
```

Comment:
We should leave out the non-dependency changes to `pyproject.toml`. E.g., the description, authors, and license

Reply:
I do not fully understand this. Aren't these needed in the .toml file?

Reply:
* The description text "Collection of methods for performing morphology appropriate representation learning" is better than the proposed "Default template for PDM package".
* The empty "{name = "", email = ""}," authorship line is unnecessary
* The license for this project is specified in the LICENSE file and is not MIT. (Beyond the scope of this PR, but the proper way to specify the license for this project is with `license = {file = "LICENSE"}`.)